src/pages/index.js

```diff
@@ -189,6 +189,7 @@ const api = new Api({
   },
 }); // log the error to the console
 
+
 let section;
 let userId;
 
```

Comment:
`".profile__image"` in `UserInfo`


Please, pass this selector into the constructor not to hardcode it inside the class


```js
export default class UserInfo {
  constructor({ name, about, avatar }) {
    this.name = document.querySelector(name);
    this.about = document.querySelector(about);
    this.avatar = document.querySelector(avatar);
  }

```